test/db/cmd/cmd_avg

```diff
@@ -0,0 +1,77 @@
+NAME=avga # add global variable
+FILE==
+CMDS=<<EOF
+avga foo 0x100 int
+avg
+avga bar 0x1000 char
+avg
+EOF
+EXPECT=<<EOF
+global int foo @ 0x100
+global int foo @ 0x100
+global char bar @ 0x1000
+EOF
+RUN
+
+NAME=avgj # output in json mode
+FILE==
+CMDS=<<EOF
+avga foo 0x100 int
+avga bar 0x1000 char
+avgj
+EOF
+EXPECT=<<EOF
+[{"name":"foo","type":"int","addr":"0x100"},{"name":"bar","type":"char","addr":"0x1000"}]
+EOF
+RUN
+
+NAME=avgd # delete global variable
+FILE==
+CMDS=<<EOF
+avga foo 0x100 int
+avga bar 0x1000 char
+avg
+avgd 0x100
+avg
+avga foofoo 0x100 int
+avg
+avgd 0x102
+avg
+EOF
+EXPECT=<<EOF
+global int foo @ 0x100
+global char bar @ 0x1000
+global char bar @ 0x1000
+global char bar @ 0x1000
+global int foofoo @ 0x100
+global char bar @ 0x1000
+EOF
+RUN
+
+NAME=avgn # rename global variable
+FILE==
+CMDS=<<EOF
+avga foo 0x100 int
+avg
+avgn foo bar
+avg
+EOF
+EXPECT=<<EOF
+global int foo @ 0x100
+global int bar @ 0x100
+EOF
+RUN
+
+NAME=avgt # retype global variable
```

Comment:
I suggest to add a couple more tests for more complex types - structures, unions, enums, function pointers.
Also, add a types of pointer and array.